dev/io.openliberty.grpc.1.0.internal.common/bnd.bnd

```diff
@@ -15,7 +15,7 @@
 -sub: *.bnd
```

Comment:
Update copyright date range to include 2023

Reply:
Done

Reply:
got missed it looks with your latest updates, should have a 2023 date range included.